GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Saturn-Skylab/RO_Saturn.cfg

```diff
@@ -0,0 +1,2112 @@
+			}
+		}
+	}
+
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = Length1
+	}
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = Length2
+	}
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = Length4
+	}
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = Length1_CB
+	}
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = Length0p5
+	}
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = Length0
+	}
+
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = Default
+		volume = 2504424.205 //5.6Mlbs capacity
+		basemass = -1
+
+		TANK
+		{
+			name = Kerosene
+			amount = 959005.275
+			maxAmount = 959005.275
+		}
+		TANK
+		{
+			name = LqdOxygen
+			amount = 1545418.93
+			maxAmount = 1545418.93
+		}
+	}
+}
+
++PART[bluedog_Saturn_S1C_Tank]:FOR[RealismOverhaul]
+{
+	@name = bluedog_Saturn_MS1C1A_Tank
+	%RSSROConfig = True
+	@rescaleFactor = 1.5529
+	@mass = 89.523 //an increase of 56,000 lbs in dry weight
+	@maxTemp = 673.15
+
+	@real_title = MLV Saturn V MS-IC-1A Tank
+
+	!RESOURCE[],*{}
+	!MODULE[ModuleB9PartSwitch]:HAS[#moduleID[meshSwitchLength]]{}
+
+	MODULE
+	{
+		name = ModuleB9PartSwitch
+		moduleID = meshSwitchLength
+		switcherDescription = Length
+		switcherDescriptionPlural = Lengths
+		affectDragCubes = True
+		affectFARVoxels = True
+		parentID = fuelSwitch
+
+		SUBTYPE
+		{
+			name = MS-1C
+			transform = Length3
+		}
+	}
+
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = Length1
+	}
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = Length2
+	}
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = Length4
+	}
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = Length1_CB
+	}
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = Length0p5
+	}
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = Length0
+	}
+
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = Default
+		volume = 2504424.205 //5.6Mlbs capacity
+		basemass = -1
+
+		TANK
+		{
+			name = Kerosene
+			amount = 959005.275
+			maxAmount = 959005.275
+		}
+		TANK
+		{
+			name = LqdOxygen
+			amount = 1545418.93
+			maxAmount = 1545418.93
+		}
+	}
+}
+
+@PART[bluedog_Saturn_S1D_BoosterSkirt]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.5529
+	@mass = 10.5 //guess
+
+	@MODULE[ModuleEnginesFX]
+	{
+		@minThrust = 204.3
+		@maxThrust = 204.3
+
+		@atmosphereCurve
+		{
+			@key = 0 251
+			@key = 1 214
+		}	
+	}
+
+	@RESOURCE[SolidFuel]
+	{
+		%amount = 500 //guess
+		%maxAmount = 500
+	}
+}
+
+@PART[bluedog_Saturn_S1D_Parachute]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.5529
+	@mass = 0.35 //guess
+}
+
+@PART[bluedog_Saturn_S1D_SustainerTank]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.5529
+	@mass = 16.5 //guess
+}
+
+@PART[bluedog_Saturn_S1D_Vernier]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.5529
+
+	!MODULE[ModuleAlternator],*{}
+	@MODULE[ModuleGimbal],*{}
```

Comment:
The error is here. Not sure what it's suppsoed to be

Reply:
It might be the additional {} in line 697. I have removed it, hopefully that fixes it